tests/app/delivery/test_send_to_providers.py

```diff
@@ -511,18 +524,36 @@ def test_get_html_email_renderer_prepends_logo_path(notify_api):
         name='Justice League',
         text='League of Justice',
     )
+    organisation = Organisation(
+        name='some org name'
+    )
     service = Service(
         email_branding=email_branding,
+        name='some service name',
+        organisation=organisation
+    )
+    template = Template(
+        id='tempid',
+        name='some template name'
+    )
+    notification = Notification(
+        id='notid',
+        subject='email subject',
+        service=service,
+        template=template
     )
 
-    renderer = send_to_providers.get_html_email_options(service)
+    renderer = send_to_providers.get_html_email_options(notification, app.govdelivery_client)
```

Comment:
Please use a mock provider to not tie tests to any of the real providers.

Reply:
Used the mock provider defined the root conftest